javascript.js

```diff
@@ -11,4 +11,288 @@ document.getElementById('open-menu').onclick = openNav;
+  tecnologies: ['HTML', 'Bootstrap', 'Ruby'],
+  image: 'url(src/bg-project.svg)',
+  imagepopup: 'src/project-popup.svg',
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and 
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy 
+  text ever since the 1500s, when an unknown printer took a galley of type 
+  and scrambled it to make a type specimen book. It has survived not only 
+  five centuries, but also the leap into electronic typesetting, remaining essent`,
+  live: '#',
+  source: '#',
+};
+
+const project5 = {
+  title: 'Profesional Art Printing Data',
+  paragraph: `A daily selection of privately personalized reads;
+   no accounts or sign-ups required. has been the industry's standard`,
+  tecnologies: ['HTML', 'Bootstrap', 'Ruby'],
+  image: 'url(src/bg-project.svg)',
+  imagepopup: 'src/project-popup.svg',
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and 
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy 
+  text ever since the 1500s, when an unknown printer took a galley of type 
+  and scrambled it to make a type specimen book. It has survived not only 
+  five centuries, but also the leap into electronic typesetting, remaining essent`,
+  live: '#',
+  source: '#',
+};
+
+const recentWorks = {
+  image: 'src/Img%20Placeholder.svg',
+  imagepopup: 'src/project-popup.svg',
+  title: 'Multi-Post Stories',
+  paragraph: `A daily selection of privately personalized reads; 
+  no accounts or sing-ups required. has been the industry's standard 
+  dummy text ever since the 1500s, when an unknown printer took a standerd dummy text.`,
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and typesetting 
+  industry. Lorem Ipsum has been the industry's standard dummy text ever since the 
+  1500s, when an unknown printer took a galley of type and scrambled it 1960s with the
+   releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+    Ipsum has been the industry's standard dummy text ever since the 1500s, when an 
+    unknown printer took a galley of type and scrambled it 1960s with the releax map 
+    lapora verita.`,
+  tecnologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  live: '#',
+  source: '#',
+};
+
+const projectsArray = [
+  project0,
+  project1,
+  project2,
+  project3,
+  project4,
+  project5,
+  recentWorks,
+];
+
+function popup(num) {
+  document.getElementById('popup').innerHTML = `
+  <div class="darkpage"></div>
+  <div class="popup-container" >
+    <button id="close-popup" class="close-pu" type="button">&times;</button>
+    <h3 class="works-title-2 title-pu">${projectsArray[num].title}</h3>
+    <ul class="ul-p ul-pu">  
+      <li class="li-p li-pu">${projectsArray[num].tecnologies[0]}</li>  
+      <li class="li-p li-pu">${projectsArray[num].tecnologies[1]}</li>  
+      <li class="li-p li-pu">${projectsArray[num].tecnologies[2]}</li>
```

Comment:
- [ ] [OPTIONAL] Hi, in a situation like these you can avoid hardcoding the values by using the `map` function to iterate through however many entries are in the array. that way it will be more scalable and less error-prone. In a situation where you have to projects having a different number of project technologies, your code might crash. looping through the `technologies` will ensure that this code could always run regardless of how many technologies they are